packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.spec.ts

```diff
@@ -0,0 +1,524 @@
+                    { key: '2', label: 'alpha', defaultMapping: true },
+                    { key: '3', label: 'alpha' },
+                    { key: '4', label: 'alpha', defaultMapping: true }
+                ],
+                'number'
+            ));
+            await connect();
+            await waitForUpdatesAsync();
+            const column = element.columns[0] as TableColumnEnumText;
+            expect(column.checkValidity()).toBeFalse();
+            expect(column.validity.multipleDefaultMappings).toBeTrue();
+        });
+
+        describe('invalid mappings', () => {
+            // prettier-ignore
+            async function setupInvalidMappings(): Promise<Fixture<Table<SimpleTableRecord>>> {
+                return fixture<Table<SimpleTableRecord>>(
+                    html`<${tableTag} style="width: 700px">
+                            <${tableColumnEnumTextTag} field-name="field1">
+                                Column 1
+                                <${mappingTextTag} key="foo" label="foo"></${mappingTextTag}>
+                                <${mappingIconTag} key="bar" label="bar" icon="nimble-icon-xmark"></${mappingIconTag}>
+                            </${tableColumnEnumTextTag}>
+                            <${tableColumnEnumTextTag} field-name="field1">
+                                Column 2
+                                <${mappingTextTag} key="foo" label="foo"></${mappingTextTag}>
+                                <${mappingSpinnerTag} key="bar" label="bar"></${mappingSpinnerTag}>
+                            </${tableColumnEnumTextTag}>
+                        </${tableTag}>`
+                );
+            }
+            it('is invalid with icon or spinner mappings', async () => {
+                ({ element, connect, disconnect } = await setupInvalidMappings());
+                await connect();
+                await waitForUpdatesAsync();
+                const column1 = element.columns[0] as TableColumnEnumText;
+                const column2 = element.columns[1] as TableColumnEnumText;
+                expect(column1.checkValidity()).toBeFalse();
+                expect(column1.validity.unsupportedMappingType).toBeTrue();
+                expect(column2.checkValidity()).toBeFalse();
+                expect(column2.validity.unsupportedMappingType).toBeTrue();
+            });
+        });
+
+        it('catches duplicate key values', async () => {
+            ({ element, connect, disconnect } = await setup([
+                { key: 'a', label: 'alpha' },
+                { key: 'a', label: 'alpha' }
+            ]));
+            await connect();
+            await waitForUpdatesAsync();
+            const column = element.columns[0] as TableColumnEnumText;
+            expect(column.checkValidity()).toBeFalse();
+            expect(column.validity.duplicateMappingKey).toBeTrue();
+        });
+
+        it('catches equivalent numeric key values', async () => {
```

Comment:
Is there a reason you switch between `it` statements of "catches..." and "is invalid with..."? This seems like it would have been just fine as "is invalid with equivalent numeric key values". The reasoning behind the consistency is when I read a differently phrased `it` statement (when it didn't need to be), my mind had to do a double-take to see what it was you were asserting.

Reply:
Changed them to "is invalid with..."